### PR TITLE
DSS-582: only return _source info on 'raw' output_format

### PR DESCRIPTION
**Pull request by mweiden**
Fixes https://github.com/HumanCellAtlas/data-store/issues/582
Connects to https://waffle.io/HumanCellAtlas/data-store/cards/59f0d13a29243f01cc41eb1d

### Test plan
TODO: The absence of the `metadata` field in non-raw output queries is tested for. The presence of `_source` information in the `metadata` field is tested for in raw output queries.

### CODE REVIEW

**Comment by Bento007** [comment]
as a naming convention our branches usually start with your username, for example: `mweiden-582-source`

File: tests/test_search.py [hunk 1]
```diff
@@ -99,6 +110,7 @@ def test_search_returns_no_results_when_no_documents_indexed(self):
     def test_search_returns_no_result_when_query_does_not_match_indexed_documents(self):
         query = \
             {
+                "_source": False,
```
Comment (Bento007) [inline tests/test_search.py]: why did you add this?

**Comment by mweiden** [comment]
@Bento007 ty
